Makefile

```diff
@@ -18,6 +18,11 @@ SILENT ?= @
 #  usage: "path/to/ignored|another/path"
 UNIT_TEST_IGNORE := "stackrox/rox/sensor/tests"
 
+# OUTSIDE_MODULE specifies the pattern of files which are not in the main module.
+# the pattern is passed to: grep -Ev
+#  usage: "pattern/to/ignored|another/pattern"
+OUTSIDE_MODULE := "^\./scanner/"
```

Comment:
This was added because we tried to `go list ./scanner/<path>` when running unit tests, but this directory is in a separate go.mod, so go list failed

Reply:
@RTann I think I would call it UNIT_TEST_SEPARATELY